src/startupscriptgenerator/src/ruby/scriptgenerator.go

```diff
@@ -38,7 +38,7 @@ func (gen *RubyStartupScriptGenerator) GenerateEntrypointScript() string {
 
 	enableDynamicInstall := common.GetBooleanEnvironmentVariable(consts.EnableDynamicInstallKey)
 	if enableDynamicInstall && !common.PathExists(rubyInstallationRoot) {
-		scriptBuilder.WriteString(fmt.Sprintf("oryx setupEnv -appPath %s\n", gen.SourcePath))
+		scriptBuilder.WriteString(fmt.Sprintf("oryx prep --src %s\n", appPath))
```

Comment:
why? i would not change this  for now

Reply:
Same as https://github.com/microsoft/Oryx/pull/1149/files/ce4f94e5a8fe1c05e3865f38118613f56f67d30d#r745944937